client/tests/test_model_registry.py

```
@@ -0,0 +1,143 @@
+import json
+import shutil
+import uuid
+
+import pytest
```

Flake8 (7.2.0)

[error] 'pytest' imported but unused

(F401)